### PR TITLE
fix(restInterface.js): Hide metrics link unless user is an app mall stew...

### PR DESCRIPTION
**Pull request by SOAP1914**
...ard.

metrics issue #490

### CODE REVIEW

File: src/app/ozpToolbar/ozpToolbar.js [hunk 1]
```diff
@@ -152,6 +152,23 @@ var app = angular.module( 'ozpWebtop.ozpToolbar')
...
+       /**
+      * @property isAdmin indicates if the metrics link in ozpToolbar should be hidden
+      * @param board the changed board object
+      * @returns {isAdmin}
+      */
+
+     restInterface.getProfile().then(function(d){
+       $scope.userRole=d.highestRole;
+        
+       var isAdmin = $scope.userRole;
```
Comment (arw180) [inline src/app/ozpToolbar/ozpToolbar.js]: this variable should be called something else, like `userRole`. Also, this logic sets `$scope.isAdmin = true` if the user's role is `USER`. Is that really correct?

File: src/app/ozpToolbar/ozpToolbar.js [hunk 2]
```diff
@@ -152,6 +152,23 @@ var app = angular.module( 'ozpWebtop.ozpToolbar')
...
+       if(isAdmin === 'USER'){
```
Comment (SOAP1914) [inline src/app/ozpToolbar/ozpToolbar.js]: Hey Alan but  if(isAdmin === 'USER') than this is true That's what I had come back to me in the api output. and do you think I should switch "the person" isAdmin to "the person " userRole being I'm already $scoping it. if so I'll change it asap. Thanks again for the comment

File: src/app/ozpToolbar/ozpToolbar.js [hunk 3]
```diff
@@ -152,6 +152,23 @@ var app = angular.module( 'ozpWebtop.ozpToolbar')
       });
     };
 
+       /**
+      * @property isAdmin indicates if the metrics link in ozpToolbar should be hidden
+      * @param board the changed board object
```
Comment (arw180) [inline src/app/ozpToolbar/ozpToolbar.js]: remove the `@param` and `@return` comments if you're not going to create a function here

File: src/app/ozpToolbar/ozpToolbar.js (hunk 2 above)
Comment (arw180) [inline src/app/ozpToolbar/ozpToolbar.js]: In my mind, I would think that this:
```
restInterface.getProfile().then(function(d){
       $scope.userRole=d.highestRole;
        
       var isAdmin = $scope.userRole;
       if(isAdmin === 'USER'){
          $scope.isAdmin =  true;
       }else{
        $scope.isAdmin = false;
      }
    }); 
```
should really be more like:
```
restInterface.getProfile().then(function(d){
       var userRole=d.highestRole;
       if(userRole === 'ADMIN'){
          $scope.isAdmin =  true;
       }else{
        $scope.isAdmin = false;
      }
}); 
```
Does that make sense?

Comment (SOAP1914) [inline src/app/ozpToolbar/ozpToolbar.js]: Make more sense now, being I now know I'm not a Admin user :) lol
Thanks!

File: src/app/ozpToolbar/ozpToolbar.js [hunk 4]
```diff
@@ -202,6 +202,19 @@ var app = angular.module( 'ozpWebtop.ozpToolbar')
       });
     };
 
+       /**
+      * @property isAdmin indicates if the metrics link in ozpToolbar should be hidden
+      */
+
+     restInterface.getProfile().then(function(d){
+       var userRole=d.highestRole;
+       if(userRole === 'ADMIN'){
```
Comment (arw180) [inline src/app/ozpToolbar/ozpToolbar.js]: As per Clark's last comment on this issue, also accept a user role of `METRICS`: i.e. `if(userRole === 'ADMIN' || userRole === 'METRICS') {...}`

**Comment by arw180** [comment]
It looks like you merged this branch with the latest `develop` instead of rebasing it. Please rebase feature branches instead of merging them! In almost all cases, the commits contained in your pull request should be yours and yours alone - as you can see, this PR now has one of Sarah's commits. Git's smart and this won't break anything, but it uglifies the history and makes it harder for reviewers to inspect your changes.

**Pull request closed without merging**